### PR TITLE
Einstellungsicon in UC und mit Settings verlinken

### PR DESCRIPTION
**Pull request by luisafuss**
https://pw-netz.atlassian.net/browse/TINF-428 und
https://pw-netz.atlassian.net/browse/TINF-448

### CODE REVIEW

File: SAKPaaS/src/app/components/sidenav/sidenav.component.html [hunk 1]
```diff
@@ -31,16 +31,25 @@
...
+  <div class="space"></div>
+  <div class="bottom">
+    <div class="section"  *ngIf="authService.getUser() | async">
       <a routerLink="/" (click)="close()" [queryParams]="{showBadges: true}" *ngIf="authService.getUser() | async">
         <div class="img-wrap">
           <mat-icon>emoji_events</mat-icon>
         </div>
         Badges
       </a>
+      <a rel="noreferrer" href="https://auth.happyhamster.org/auth/realms/happyhamster/account/"  *ngIf="authService.getUser() | async">
```
Comment (lepathie) [inline SAKPaaS/src/app/components/sidenav/sidenav.component.html]: A `title` would be neat.

Comment (lepathie) [inline SAKPaaS/src/app/components/sidenav/sidenav.component.html]:
```suggestion
      <a rel="noreferrer" href="https://auth.happyhamster.org/auth/realms/happyhamster/account/"  *ngIf="authService.getUser() | async" title="{{'sidenav.settings' | translate}}">
```

File: SAKPaaS/src/app/components/user-card/user-card.component.html [hunk 2]
```diff
@@ -5,10 +5,16 @@
     <div class="expand">
       <mat-icon (click)="change()" class="expand-icon">keyboard_arrow_left</mat-icon>
     </div>
+    
+    <div *ngIf="expanded">
+      <a href="https://auth.happyhamster.org/auth/realms/happyhamster/account/">
```
Comment (lepathie) [inline SAKPaaS/src/app/components/user-card/user-card.component.html]:
```suggestion
      <a href="https://auth.happyhamster.org/auth/realms/happyhamster/account/" title="title="{{'sidenav.settings' | translate}}"">
```

and in addition pls add it to the text-symbols. :)

File: SAKPaaS/src/assets/i18n/de.json [hunk 3]
```diff
@@ -164,7 +164,8 @@
     "social-media": "Besuche uns auf",
     "feedback": "Feedback geben",
     "toggle-lang": "Switch to English",
-    "search": "Ladensuche"
+    "search": "Ladensuche",
+    "settings": "Einstellungen"
```
Comment (lepathie) [inline SAKPaaS/src/assets/i18n/de.json]:
```suggestion
    "settings": "Benutzer-Einstellungen"
```

Comment (lepathie) [reply SAKPaaS/src/assets/i18n/de.json]: Match the translations, pls.

File: SAKPaaS/src/assets/i18n/en.json [hunk 4]
```diff
@@ -164,7 +164,8 @@
     "social-media": "Visit us on",
     "feedback": "Give feedback",
     "toggle-lang": "Zu Deutsch wechseln",
-    "search": "Search"
+    "search": "Search",
+    "settings": "Usersettings"
```
Comment (lepathie) [inline SAKPaaS/src/assets/i18n/en.json]:
```suggestion
    "settings": "User settings"
```

**Review by lepathie — CHANGES_REQUESTED** [review]

File: SAKPaaS/src/assets/i18n/de.json [hunk 5]
```diff
@@ -164,7 +164,8 @@
     "social-media": "Besuche uns auf",
     "feedback": "Feedback geben",
     "toggle-lang": "Switch to English",
-    "search": "Ladensuche"
+    "search": "Ladensuche",
```
Comment (lepathie) [inline SAKPaaS/src/assets/i18n/de.json]:
```suggestion
    "search": "Suche",
```

**Review by lepathie — CHANGES_REQUESTED** [review]

File: SAKPaaS/src/app/components/user-card/user-card.component.html (hunk 2 above)
Comment (lepathie) [reply SAKPaaS/src/app/components/user-card/user-card.component.html]: That copy'n'taste error I did here. ^^

Comment (luisafuss) [reply SAKPaaS/src/app/components/user-card/user-card.component.html]: Already fixed

File: SAKPaaS/src/app/components/sidenav/sidenav.component.html [hunk 6]
```diff
@@ -31,16 +31,25 @@
...
+    <div class="section"  *ngIf="authService.getUser() | async">
       <a routerLink="/" (click)="close()" [queryParams]="{showBadges: true}" *ngIf="authService.getUser() | async">
         <div class="img-wrap">
           <mat-icon>emoji_events</mat-icon>
         </div>
         Badges
       </a>
+      <a rel="noreferrer" href="https://auth.happyhamster.org/auth/realms/happyhamster/account/"  *ngIf="authService.getUser() | async" title="{{'sidenav.settings' | translate}}">
+        <div class="img-wrap">
+          <mat-icon fontSet="material-icons" class="settings-icon">settings</mat-icon>
```
Comment (lepathie) [inline SAKPaaS/src/app/components/sidenav/sidenav.component.html:48]: image

Icon is outside container.

File: SAKPaaS/src/app/components/user-card/user-card.component.scss [hunk 7]
```diff
@@ -14,10 +14,9 @@
     cursor: pointer;
   }
   .settings-icon {
-    display: none;
+    color: var(--midnight-blue);
     float: right;
-    padding-left: 10px;
-    width: 34px;
+    width: 19px;
```
Comment (lepathie) [inline SAKPaaS/src/app/components/user-card/user-card.component.scss]:
```suggestion
```
should remove that.

File: SAKPaaS/src/app/components/user-card/user-card.component.scss [hunk 8]
```diff
@@ -14,10 +14,9 @@
     cursor: pointer;
   }
   .settings-icon {
-    display: none;
+    color: var(--midnight-blue);
     float: right;
```
Comment (lepathie) [inline SAKPaaS/src/app/components/user-card/user-card.component.scss]: Personally I'd prefer:
```suggestion
    position: absolute;
    right: 0;
```

Just makes more sense imo.

Comment (lepathie) [reply SAKPaaS/src/app/components/user-card/user-card.component.scss]: You have to add `position: relative;` to `#user-card .content` for it to work though.

File: SAKPaaS/src/app/components/user-card/user-card.component.scss (hunk 7 above)
Comment (lepathie) [reply SAKPaaS/src/app/components/user-card/user-card.component.scss]: See earlier comment for reason.

**Review by lepathie — CHANGES_REQUESTED** [review]
image
The icon is outside the container.

**Review by lepathie — APPROVED** [review]
Now I'm happy. 🙂

**Review by florianredmann — APPROVED** [review]

**Pull request merged**